cli/cmd/vuln_container.go

```diff
@@ -754,3 +804,93 @@ func filterSeverity(severity string, threshold string) bool {
 func vulFiltersEnabled() bool {
 	return vulCmdState.Severity != "" || vulCmdState.Fixable
 }
+
+func getContainerRegistries() ([]string, error) {
+	var (
+		registries            = make([]string, 0)
+		regsIntegrations, err = cli.LwApi.Integrations.ListContainerRegistries()
+	)
+	if err != nil {
+		return registries, errors.Wrap(err, "unable to get container registry integrations")
+	}
+
+	for _, i := range regsIntegrations.Data {
+		// avoid adding empty registries coming from the new local_scanner
+		if i.Data.RegistryDomain == "" {
+			continue
+		}
+
+		// avoid adding duplicate registries
+		if array.ContainsStr(registries, i.Data.RegistryDomain) {
+			continue
+		}
+
+		registries = append(registries, i.Data.RegistryDomain)
+	}
+
+	return registries, nil
+}
+
+// Creates a user-friendly error message
+func userFriendlyErrorForOnDemandCtrVulnScan(err error, registry, repo, tag string) error {
+	if strings.Contains(err.Error(),
+		"Could not find integraion matching the registry provided",
```

Comment:
typo

```suggestion
		"Could not find integration matching the registry provided",
```

Reply:
Unfortunately, this is a typo in the platform, so we have to add the same typo in our code 😭

Reply:
:(